src/access_nri_intake/utils.py

```diff
@@ -102,7 +102,21 @@ def validate_against_schema(instance: dict, schema: dict) -> None:
         Validator, type_checker=type_checker
     )
 
-    TupleAllowingValidator(schema).validate(instance)
+    issues = list(TupleAllowingValidator(schema).iter_errors(instance))
+
+    if len(issues) > 0:
+        issue_str = ""
+        for i, issue in enumerate(issues):
+            try:
+                issue_str += (
+                    f"\n{i+1:02d} | {issue.absolute_path[0]} : { issue.message }"
+                )
+            except IndexError:  # Must be a missing keyword, not a bad type/value
+                issue_str += f"\n{i+1:02d} | (missing) : { issue.message }"
+        # raise jsonschema.ValidationError(textwrap.fill(textwrap.dedent(issue_str)))
```

Comment:
I think this commented out line is redundant? Looks like WIP to me

Reply:
Yep, removed